march_rqt_input_device/src/march_rqt_input_device/input_device.py

```diff
@@ -129,6 +129,14 @@ def __init__(self, context):
                                               text='MV Ramp and Door slope up',
                                               callback=lambda: self.publish_gait('gait_ramp_door_slope_up'))
 
+        gait_ramp_door_slope_down = MarchButton(name='gait_ramp_door_slope_down',
+                                                text='MV Ramp and Door slope down',
+                                                callback=lambda: self.publish_gait('gait_ramp_door_slope_down'))
+
+        gait_ramp_door_last_step = MarchButton(name='gait_ramp_door_last_step',
+                                               text='MV Ramp and Door last step',
```

Comment:
You can name them `Slope down` and `Slope last step`, since these are not bound to MV gaits